pkg/controllers/vdb/createdb_reconciler.go

```diff
@@ -175,7 +175,7 @@ func (c *CreateDBReconciler) generatePostDBCreateSQL(ctx context.Context, initia
 	// config parm in the create db call.
 	if c.Vdb.Spec.EncryptSpreadComm != vapi.EncryptSpreadCommDisabled && ok && vinf.IsOlder(vapi.SetEncryptSpreadCommAsConfigVersion) {
 		sb.WriteString(fmt.Sprintf(`alter database default set parameter EncryptSpreadComm = '%s';
-		`, c.Vdb.Spec.EncryptSpreadComm))
+		`, vapi.EncryptSpreadCommWithVertica))
```

Comment:
Why did you do this? What if a user wants to disable spread encryption(if that even makes sense)?

Reply:
The code here is for enabling spread encryption. The if statement above makes sure the user does not want to disable spread encryption. When encryptSpreadComm is an empty string, 'c.Vdb.Spec.EncryptSpreadComm' won't give the correct value to the ddl, but 'vapi.EncryptSpreadCommWithVertica' will.